tests/e2e/api/transactionResponse.test.ts

```diff
@@ -0,0 +1,530 @@
+                  },
+                ],
+              },
+            },
+          },
+          type: "write_resource",
+        },
+      ],
+      events: [
+        {
+          guid: {
+            creation_number: "0",
+            account_address: "0x0",
+          },
+          sequence_number: "0",
+          type: "0x1::jwks::ObservedJWKsUpdated",
+          data: {
+            epoch: "2",
+            jwks: {
+              entries: [
+                {
+                  issuer: "0x68747470733a2f2f6163636f756e74732e676f6f676c652e636f6d",
+                  jwks: [
+                    {
+                      variant: {
+                        data: "0x2835356331383861383335343666633138386535313537366261373238333665303630306538623733035253410552533235360441514142d60271304372463378336159736a7230594f4c4d4f416845474d76794670366f34527179456455726e5444596b685a626375642d664a4551616643546e6a533951484e31496a70754b366770783569332d5a363376526a7335455158376c50316a473851672d436e426454544c7734754a6937526d6d6c4b507359614f3144624e6b464f3275454e3632734f4f7a6d4a4368316f6433435a5849315559483563765a5f734c4a614e3241345477765554553361586c5862554e4a7a5f4879336c3071314a6a746137354e724a744a3750666a3974565873387158703135745a58726e62614d2d41493070757377743335567351626d4c77556f7646464765546f6f357132635f633178596e56357551594d6164414e656b47505246504d394a5a705353497648304c765f66313556327a52716d49675837613352636d546e72332d7733514e51546f6764792d4d6f67785055645262786f77",
+                        type_name: "0x1::jwks::RSA_JWK",
+                      },
+                    },
+                    {
+                      variant: {
+                        data: "0x2865643830366631383432623538383035346231386236363964643161303961346633363761666334035253410552533235360441514142d60272483351354e59364d41656145384e75537737527732436331655f6a2d6b555330343474752d57636d54467a424b54754b76496c676a357730536c536269566c38317a4274657451467475776b4d7a57676e436b732d322d4677706f795f5f324e556f7555674c74496767415645794f4767504c6679617377746b536d5a73556d575767394a3843674d55646f58466b625a41506c616444636d537169584a376370396e76726f366634736a66724744597a355f2d534e7a314151454762766354683945655a6b764b50726d6e56335945523935624a73676b486d4e4a566b51364c6357744c794b6853475147524d655459615844616a63324b724b54336e657437714e6862416d374b70576464627452356c36413054524372414d6f56324d36385f474c5246323461636a33554f35525730536b756142545a53344b5170796f7941424341746a4c53722d335259365752396e7077",
+                        type_name: "0x1::jwks::RSA_JWK",
+                      },
+                    },
+                  ],
+                  version: "1",
+                },
+              ],
+            },
+          },
+        },
+      ],
+      timestamp: "1708575651139233",
+      type: TransactionResponseType.Validator,
+    } satisfies ValidatorTransactionResponse;
+  });
+
+  test("it tests a sample block_metadata JSON response satisfies ValidatorTransactionResponse", async () => {
```

Comment:
This name is misleading, since it's checking for block metadata to look like block metadata.